core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2678,6 +2696,7 @@ public String toString() {
           .setDescription("The maximum number of workers to retry before the client gives up on "
               + " reading a block")
           .build();
+  // TODO(cc): remove this since configuration propagation is always enabled?
```

Comment:
did we decide to use a single property for refresh mode: never, once or using heartbeat?

Reply:
for path conf, if the mode is never or once, the path conf feature is of no use to clients?

Reply:
makes sense